data/projects.yml

```diff
@@ -0,0 +1,16 @@
+title: Projects
+introduction: |
+  These are the open source projects which are officially hosted by the
+  OpenRail Association. They have been accepted through the incubation process.
+  If you would like to suggest a new project to the OpenRail Association please
+  have a look at the questionnaire for new projects.
```

Comment:
I would link to the incubation process and the questionnaire if we already mention them here.

Reply:
Good idea.